extras.go

```diff
@@ -74,6 +75,38 @@ func addExtrasRoutes(r chi.Router) {
 		format.JSON(w, 200, Response{ID: id})
 	})
 
+	r.Post("/tags", func(w http.ResponseWriter, r *http.Request) {
+		r.ParseForm()
+		name := r.Form.Get("name")
+		color := r.Form.Get("color")
+		value := strings.ReplaceAll(name, " ", "")
+
+		res, _ := conn.Exec("INSERT INTO tag (name, value, color) VALUES (?, ?, ?)", name, value, color)
+
+		cid, _ := res.LastInsertId()
+		format.JSON(w, 200, Response{ID: strconv.FormatInt(cid, 10), Value: value})
```

Comment:
I think we need to use the specific response type here instead of adding the ID to the common Response type

Reply:
both add and update methods return full TagInfo now